src/System.Private.DataContractSerialization/src/System/Xml/XmlNodeWriter.cs

```diff
@@ -10,25 +10,41 @@
 using System.Globalization;
 using System.Runtime.Serialization;
 using System.Collections.Generic;
-
+using System.Threading.Tasks;
 
 namespace System.Xml
 {
     internal abstract class XmlNodeWriter
     {
         public abstract void Flush();
+        public virtual Task FlushAsync()
```

Comment:
Why are these virtual rather than abstract?  Since the type is internal, can't we ensure that all derived types override them appropriately?